internal/config.go

```diff
@@ -4,42 +4,116 @@ import (
 	"github.com/spf13/viper"
 	"github.com/wal-g/wal-g/internal/tracelog"
 	"os"
-	"strings"
+)
+
+const (
+	DownloadConcurrencySetting   = "WALG_DOWNLOAD_CONCURRENCY"
+	UploadConcurrencySetting     = "WALG_UPLOAD_CONCURRENCY"
+	UploadDiskConcurrencySetting = "WALG_UPLOAD_DISK_CONCURRENCY"
+	UploadQueueSetting           = "WALG_UPLOAD_QUEUE"
+	SentinelUserDataSetting      = "WALG_SENTINEL_USER_DATA"
+	PreventWalOverwriteSetting   = "WALG_PREVENT_WAL_OVERWRITE"
+	DeltaMaxStepsSetting         = "WALG_DELTA_MAX_STEPS"
+	DeltaOriginSetting           = "WALG_DELTA_ORIGIN"
+	CompressionMethodSetting     = "WALG_COMPRESSION_METHOD"
+	DiskRateLimitSetting         = "WALG_DISK_RATE_LIMIT"
+	NetworkRateLimitSetting      = "WALG_NETWORK_RATE_LIMIT"
+	UseWalDeltaSetting           = "WALG_USE_WAL_DELTA"
+	LogLevelSetting              = "WALG_LOG_LEVEL"
+	TarSizeThresholdSetting      = "WALG_TAR_SIZE_THRESHOLD"
+	GpgKeyIDSetting              = "GPG_KEY_ID"
+	PgpKeySetting                = "WALG_PGP_KEY"
+	PgpKeyPathSetting            = "WALG_PGP_KEY_PATH"
+	PgpKeyPassphraseSetting      = "WALG_PGP_KEY_PASSPHRASE"
```

Comment:
"Setting" suffix looks spare. You use it to getting information that it comes from env variables.
It would be more explicit to create a struct Environment {}. 
For example, use it as env.DownloadConcurrency.